containerm/deployment/deployment.go

```diff
@@ -87,10 +88,10 @@ func (d *deploymentMgr) Deploy(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		if !entry.IsDir() {
+		if !entry.IsDir() && strings.HasSuffix(path, ".json") {
 			ctr, readErr := util.ReadContainer(path)
 			if readErr != nil {
-				log.ErrorErr(readErr, "error reading container configuration from file = %s", path)
+				log.ErrorErr(readErr, "error reading container configuration from file =", path)
```

Comment:
```suggestion
				log.ErrorErr(readErr, "error reading container configuration from file = %s", path)
```
I guess this was deleted by mistake. It breaks the error formatting.